### PR TITLE
Move to Github Actions

### PR DESCRIPTION
**Pull request by sidz**
Move to GitHub Actions instead of travis and prettyCI.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,16 +32,16 @@
         "composer-plugin-api": "^1.1 || ^2.0"
     },
     "require-dev": {
-        "composer/composer": "^1.9",
-        "friendsofphp/php-cs-fixer": "^2.16",
+        "composer/composer": "^1.9 || ^2.0",
+        "friendsofphp/php-cs-fixer": "^2.18, <2.19",
```
Comment (sidz) [inline composer.json:36]: cs fixer 2.19 shows a lot of deprecation messages and it will be updated in the separate PR

**Review by maks-rafalko — APPROVED** [review]
